lib/rubygems/platform.rb

```diff
@@ -75,6 +75,7 @@ def initialize(arch)
 
       @cpu = case cpu
              when /i\d86/ then 'x86'
+             when /universal\.x86_64/ then 'x86_64'
```

Comment:
It might be useful to capture the architecture out of `cpu`. For an arm based Mac the RUBY_PLATFORM is `universal.arm64e-darwin20`. See [here](https://github.com/ffi/ffi/issues/860#issuecomment-748541442).

Reply:
Nice, that makes sense, thanks!

Reply:
Updated!